nsf_init -> gstnsf_init to prevent symbol conflict with security/nss

$OpenBSD: patch-gst_nsf_gstnsf_c,v 1.4 2012/03/05 15:22:46 ajacoutot Exp $
--- gst/nsf/gstnsf.c.orig	Fri Dec 30 19:24:33 2011
+++ gst/nsf/gstnsf.c	Mon Mar  5 14:44:23 2012
@@ -168,7 +168,7 @@ gst_nsfdec_class_init (GstNsfDec * klass)
   GST_DEBUG_CATEGORY_INIT (nsfdec_debug, "nsfdec", 0,
       "NES sound file (nsf) decoder");
 
-  nsf_init ();
+  gstnsf_init ();
 }
 
 static void
